component/attr_cache/attr_cache.go

```diff
@@ -168,38 +169,17 @@ func (ac *AttrCache) OnConfigChange() {
 // That way if a request came in for a deleted item, we can respond from the cache.
 func (ac *AttrCache) deleteDirectory(path string, time time.Time) {
 	// Delete all descendants of the path, then delete the path
```

Comment:
This comment is obsolete, and so is the one below (on 176) that says basically the same thing. Now that we have these slick recursive functions, we don't have to think about children here at all.

Reply:
this interface is not very good at showing exactly what you're referring to. What comments need to be deleted? If you quote a couple, that will help since line numbers change.

Reply:
`	// Delete all descendants of the path, then delete the path`
and
`// delete the path itself and children.`